core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -525,244 +559,238 @@ private String buildDeterministicValue(String key, String fieldkey) {
     return sb.toString();
   }
 
-	/**
-	 * Do one insert operation. Because it will be called concurrently from multiple client threads, this 
-	 * function must be thread safe. However, avoid synchronized, or the threads will block waiting for each 
-	 * other, and it will be difficult to reach the target throughput. Ideally, this function would have no side
-	 * effects other than DB operations.
-	 */
-	public boolean doInsert(DB db, Object threadstate)
-	{
-		int keynum=keysequence.nextInt();
-		String dbkey = buildKeyName(keynum);
-		HashMap<String, ByteIterator> values = buildValues(dbkey);
-		if (db.insert(table,dbkey,values).equals(Status.OK))
-			return true;
-		else
-			return false;
-	}
-
-	/**
-	 * Do one transaction operation. Because it will be called concurrently from multiple client threads, this 
-	 * function must be thread safe. However, avoid synchronized, or the threads will block waiting for each 
-	 * other, and it will be difficult to reach the target throughput. Ideally, this function would have no side
-	 * effects other than DB operations.
-	 */
-	public boolean doTransaction(DB db, Object threadstate)
-	{
-		String op=operationchooser.nextString();
-
-		if (op.compareTo("READ")==0)
-		{
-			doTransactionRead(db);
-		}
-		else if (op.compareTo("UPDATE")==0)
-		{
-			doTransactionUpdate(db);
-		}
-		else if (op.compareTo("INSERT")==0)
-		{
-			doTransactionInsert(db);
-		}
-		else if (op.compareTo("SCAN")==0)
-		{
-			doTransactionScan(db);
-		}
-		else
-		{
-			doTransactionReadModifyWrite(db);
-		}
-		
-		return true;
-	}
+  /**
+   * Do one insert operation. Because it will be called concurrently from multiple client threads,
+   * this function must be thread safe. However, avoid synchronized, or the threads will block waiting
+   * for each other, and it will be difficult to reach the target throughput. Ideally, this function would
+   * have no side effects other than DB operations.
+   */
+  public boolean doInsert(DB db, Object threadstate) {
+    int keynum = keysequence.nextInt();
+    String dbkey = buildKeyName(keynum);
+    HashMap<String, ByteIterator> values = buildValues(dbkey);
+
+    Status status;
```

Comment:
If you do this in DBWrapper instead of CoreWorkload, it'll work with arbitrary workloads (not that we have a huge variety ATM, but still).

If you'd prefer to keep this limited to the CoreWorkload, then please update the property to start with a prefix to indicate it's just the core workload (i.e. 'core_workload.').